src/list/list.directive.ts

```diff
@@ -26,7 +26,7 @@ export class List {
 	}
 
 	@HostBinding("class.bx--list--nested") get nested() {
-		return this.elementRef.nativeElement.parentElement.tagName === "LI";
+		return (this.elementRef.nativeElement.parentElement && this.elementRef.nativeElement.parentElement.tagName === "LI");
```

Comment:
Might be a bit tidier to just wrap this in an `if` statement 🤔

Reply:
I broke it out to see how it would look, like so: 
```typescript
if (this.elementRef.nativeElement.parentElement) {
	return this.elementRef.nativeElement.parentElement.tagName === "LI";
} else {
	return false;
}
``` 
The correct way of doing that would be to move all conditional checks into the `if` line:
```typescript
if (this.elementRef.nativeElement.parentElement && this.elementRef.nativeElement.parentElement.tagName === "LI") {
	return true;
} else {
	return false;
}
```
I feel like there's not much benefit as most js/ts devs are used to this pattern of checking that a property exists and is truthy.

Once you move to TypeScript@>3.7 it would look better with optional chaining since you could just do
```typescript
return this.elementRef.nativeElement.parentElement?.tagName === "LI";
```

Reply:
>Once you move to TypeScript@>3.7 it would look better with optional chaining since you could just do
> 
> ```typescript
> return this.elementRef.nativeElement.parentElement?.tagName === "LI";
> ```

That's going to be so nice 🤩 

Makes sense to me otherwise 👍